src/subscriptions.ts

```diff
@@ -165,13 +164,13 @@ export async function getSubscribedTransactions(
 
         catchupTransactions = catchupTransactions
           // Sort by transaction order
-          .sort((a, b) => a['confirmed-round']! - b['confirmed-round']! || a['intra-round-offset']! - b['intra-round-offset']!)
+          .sort((a, b) => Number(a.confirmedRound! - b.confirmedRound!) || a.intraRoundOffset! - b.intraRoundOffset!)
```

Comment:
The intra round offset stuff - need to make sure it all still works ok with this